scripts/fuzz_shell.js

```diff
@@ -179,8 +180,14 @@ var instance = new WebAssembly.Instance(new WebAssembly.Module(binary), imports)
 // Handle the exports.
 var exports = instance.exports;
 exports = Asyncify.instrumentExports(exports);
-if (exports.memory) {
-  var view = new Int32Array(exports.memory.buffer);
+
+var view;
+
+// Recreate the view. This is important both initially and after a growth.
+function refreshView() {
+  if (exports.memory) {
+    view = new Int32Array(exports.memory.buffer);
+  }
```

Comment:
It looks like is not immediately run whereas it was before.  I assume that is indended.

Reply:
Yes, I put it in a place that dominates all the uses.